packages/vue-app/template/index.js

```diff
@@ -120,7 +120,10 @@ async function createApp (ssrContext) {
         err = err || null
         app.context._errored = Boolean(err)
         err = err ? normalizeError(err) : null
-        const nuxt = this.nuxt || this.$options.nuxt
+        let nuxt = app.nuxt // to work with @vue/composition-api, see https://github.com/nuxt/nuxt.js/issues/6517#issuecomment-573280207
+        if (this) {
```

Comment:
Should we also check `if (!nuxt && this)` ? 

/cc @Atinux

Reply:
Actually I prefer to keep the behaviour with the instance to avoid breaking changes, we will refactor this part with Vue 3 anyway :)